### PR TITLE
Chris changes

### PR DESCRIPTION
**Pull request by chris-f-anderson**
comments are optional...

### CODE REVIEW

File: CalculatorTests/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+
+            // Assert
+            Assert.Equal(expected, actual);
+        }
+        [Theory]
+        [InlineData (100, 50, 2)]
+        [InlineData (50, 25, 2)]
+        [InlineData (20, 1, 20)]
+        public void DivideNumbers (int number1, int number2, int expected)
```
Comment (chipelmer) [inline CalculatorTests/UnitTest1.cs:61]: There's a suspiciously absent test cast, which is when dividing by zero. The tests should establish what the method being tested is supposed to return if given various inputs, including edge cases such as zero for a divisor

File: CalculatorWithTests/Calculator.cs [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+        {
+            return number1 / number2;
+        }
+        public int Factorial(int number1)
+        {
+            if (number1 == 0)
+            {
+                return 1;
+            }
+            return number1 * Factorial(number1 - 1);
```
Comment (chipelmer) [inline CalculatorWithTests/Calculator.cs:28]: Recursive solution, nice

**Review by chipelmer — APPROVED** [review]
Looks good, keep it up
